src/mito2/src/worker.rs

```diff
@@ -277,8 +272,8 @@ impl WorkerGroup {
     }
 }
 
-fn value_to_index(value: usize, num_workers: usize) -> usize {
-    value % num_workers
+fn region_id_to_index(id: RegionId, num_workers: usize) -> usize {
+    (id.table_id() as usize % num_workers) + (id.region_number() as usize % num_workers)
```

Comment:
table_id = 2, region_id = 3, workers# = 4 => index = 5 => out of bound

Reply:
👍